test/tlstest_helpers.c

```diff
@@ -0,0 +1,183 @@
+// SPDX-License-Identifier: Apache-2.0 AND MIT
+#include <openssl/ssl.h>
+#include <openssl/err.h>
+#include "testutil.h"
```

Comment:
If this include is still needed (?), we still have the dependency to the local `openssl` test directory.

Reply:
Removed